script/generate_opengrep_checksums.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Helper script to generate SHA256 checksums for all opengrep binaries
+# and automatically update the install_opengrep.py script
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+INSTALL_SCRIPT="${SCRIPT_DIR}/install_opengrep.py"
+
+# Check if sed supports -i without backup file (GNU vs BSD)
+if sed --version >/dev/null 2>&1; then
+    # GNU sed
+    SED_INPLACE=(sed -i)
+else
+    # BSD sed (macOS)
+    SED_INPLACE=(sed -i '')
+fi
+
+VERSION="${1:-v1.11.2}"
+
+echo "==========================================="
+echo "Opengrep Checksum Generator & Updater"
+echo "==========================================="
+echo
+echo "Version: ${VERSION}"
+echo "Target:  ${INSTALL_SCRIPT}"
+echo
+
+# Create temporary directory
+TEMP_DIR=$(mktemp -d)
+trap 'rm -rf "${TEMP_DIR}"' EXIT
+
+cd "${TEMP_DIR}"
+
+# All supported platforms
+# Note: Windows binaries have .exe in the distribution name
+# Note: Windows ARM64 is not available
+PLATFORMS=(
+    "opengrep_osx_arm64"
+    "opengrep_osx_x86"
+    "opengrep_manylinux_x86"
+    "opengrep_manylinux_aarch64"
+    "opengrep_musllinux_x86"
+    "opengrep_musllinux_aarch64"
+    "opengrep_windows_x86.exe"
+)
+
+echo "Step 1: Downloading binaries and signatures..."
+DOWNLOADED_COUNT=0
+for DIST in "${PLATFORMS[@]}"; do
+    BASE_URL="https://github.com/opengrep/opengrep/releases/download"
+    URL="${BASE_URL}/${VERSION}/${DIST}"
+    echo "  - ${DIST}..."
+
+    if curl -fsSL -o "${DIST}" "${URL}"; then
```

Comment:
Downloading the releases via the GitHub CLI would be an improvement

Reply:
this is only used to generate the checksums whenever we want to update the checksums. What are the advantages of github cli download?

Reply:
removed  for now